changes/ce/feat-12561.en.md

```diff
@@ -0,0 +1,3 @@
+Implement HTTP API to get the list of client's inflight and mqueue messages:
+ - GET /clients/{clientid}/mqueue_messages?page=1&limit=100
+ - GET /clients/{clientid}/inflight_messages?page=1&limit=100
```

Comment:
Maybe add:
1. he ordering of the messages in each page?
2. how to start the first page
3. how to continue the next pages
4. what happens if continuation is invalid

Reply:
Done